src/Token.php

```diff
@@ -65,21 +65,21 @@ public static function getTokenKindNameFromValue($kindName) {
     function jsonSerialize() {
         $kindName = $this->getTokenKindNameFromValue($this->kind);
 
-        if (!isset($GLOBALS["SHORT_TOKEN_SERIALIZE"])) {
-            $GLOBALS["SHORT_TOKEN_SERIALIZE"] = false;
+        if (!isset($GLOBALS['SHORT_TOKEN_SERIALIZE'])) {
```

Comment:
@decanus @tpunt @nikic - is there a particular convention around which quotation mark style is preferred?